api/client/samples/similar_regions/similar_region.py

```diff
@@ -72,6 +72,8 @@ def _similar_to(self, region_id, number_of_regions):
         # list as an index to preserve dimensionality of returned data
         x = self.state.data_standardized[self.state.mapping[region_id], :]
         x = x.reshape(1, -1)
+        assert number_of_regions < self.ball.data.shape[0], "num_regions must be smaller than or equal to total " \
```

Comment:
This would be easier to understand if we make the RHS  self.state.num_regions since that is what the user can more directly control and  so we don't need expose the internals of the balltree.